src/PageContent/RequestPickup/RecurringPickupRequest.js

```diff
@@ -308,213 +419,364 @@ class RecurringPickupRequest extends Component {
             });
     }
 
+    addFood(name, weight, event) {
+        if (name !== '' && weight !== '') {
+            var newElement = { foodName: name, foodWeight: weight };
+            this.setState({ foodRows: [...this.state.foodRows, newElement] });
+        }
+    }
+
     render() {
         return (
             <div className="form">
-                <form id={this.formId} onSubmit={this.createRequest}>
-                    <div className="info">
-                        <p id="form-heading">Schedule Recurring Pickup</p>
-                        {Object.keys(this.state.errors).map((error, i) => {
-                            return (
-                                <p className="error" key={i}>
-                                    {this.state.errors[error]}
-                                </p>
-                            );
-                        })}
-                        <span className="flex">
-                            <span className="grid">
-                                <label>
-                                    Start Date <span className="red">*</span>
-                                </label>
-                                <br />
-                                <input
-                                    type="date"
-                                    name="startDate"
-                                    onChange={this.handleChange.bind(
-                                        this,
-                                        'startDate'
-                                    )}
-                                    required
-                                />
-                                <br />
-                            </span>
-                            <span className="grid">
-                                <label>
-                                    {' '}
-                                    End Criteria <span className="red">*</span>
-                                </label>
-                                <br />
-                                <label className="container-smaller">
+                {this.props.type === DeliveryType.EMERGENCY ? (
```

Comment:
see comment in `RequestPickupWrapper` for extracting the UI out into separate components.